dev/MRTCore/mrt/Core/src/MRM.vcxproj

```diff
@@ -107,25 +103,15 @@
       <Optimization>Disabled</Optimization>
       <WarningVersion>18</WarningVersion>
     </ClCompile>
+    <Link>
+      <EnableCOMDATFolding>false</EnableCOMDATFolding>
```

Comment:
I turned off COMDAT folding for **Debug** builds per https://docs.microsoft.com/en-us/cpp/build/reference/opt-optimizations?view=msvc-160 as it can make debugging harder:
> ... it can change the function names that appear in stack traces. It can also make it impossible to set breakpoints in certain functions or to examine some data in the debugger, and can take you into unexpected functions when you single-step through your code.